app/views/content/train-to-be-a-teacher/promos/_events-promo-itt.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      caption: "4 July 2022",
+      heading: "Train to Teach Manchester",
+      link_target: "/events/train-to-teach-manchester-event-040722",
+      link_text: "Sign up for this event") do %>
+    <p>Hear from training providers, chat with teachers and get one-to-one advice from expert advisers.<p>
+  <% end %>
+
+  <% promo.right_section(
+        caption: "Events near you",
+        heading: "Events are a great place to ask questions") do %>
+      <p>Events are one of the best ways to learn more about your teacher training provider and course.</p>
```

Comment:
I think this might be better as something along the lines of 'meet providers and find out more about different courses' - the way it is worded at the moment sounds like you've already chosen one